test/integration/conflict_resolution.js

```diff
@@ -150,6 +150,107 @@ describe('Conflict resolution', () => {
     })
   })
 
+  const afterFullSyncBothTreeDeepEqual = async (expected) => {
+    await helpers.local.scan()
+    await helpers.remote.pullChanges()
+    await helpers.syncAll()
+    await helpers.local.scan()
+    await helpers.remote.pullChanges()
+    await helpers.syncAll()
+    should(await helpers.trees()).deepEqual({remote: expected, local: expected})
```

Comment:
Maybe the newly created `mergeSideEffects` helper would be useful here?

Reply:
This is an integration test, not a Merge unit test :)